chia/util/initial-config.yaml

```diff
@@ -545,6 +540,11 @@ wallet:
   # Interval to resend unconfirmed transactions, even if previously accepted into Mempool
   tx_resend_timeout_secs: 1800
 
+  # After n received unspent transactions, the spam filter will be enabled, which will filter out received
+  # coins with very small value. Any standard TX under xch_spam_amount is filtered
+  spam_filter_after_n_txs: 200
```

Comment:
Some have over 100k tx.

Reply:
Should we increase this @trepca ? I think increasing is fine since wallet perf is getting much better

Reply:
I missed the fact it's only counting "change" coins, which makes sense. It's probably unlikely that even a power user would have that many change coins, although I guess it depends on the select coin strategy. 
What we want to avoid is making life harder for power users, that might create a lot of change coins to enable more parallel minting / operations. These might hit the limit of what we consider "spam". /cc @geoffwalmsley

Reply:
if they are doing something like that, they can go to the config and change it right? Or use higher value coins